images/linux/scripts/installers/aws.sh

```diff
@@ -32,6 +32,20 @@ if ! command -v aws; then
     exit 1
 fi
 
+curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"
+sudo dpkg -i session-manager-plugin.deb
+
+sessionPlugin=$(session-manager-plugin)
+
+if [[ $sessionPlugin == *"was installed successfully"* ]]
+then
+    echo "$sessionPlugin"
```

Comment:
Can we put `echo` before `if` so we won't need to duplicate it

Reply:
Should i just remove this check, because if command not found it will fail ?